dev/com.ibm.ws.cdi.extension_fat/test-applications/SPIExtension.war/src/com/ibm/ws/cdi/extension/spi/test/app/CrossWireTestServlet.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.extension.spi.test.app;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.enterprise.inject.UnsatisfiedResolutionException;
+import javax.inject.Inject;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.ibm.ws.cdi.extension.spi.test.bundle.UnregisteredBean;
+import com.ibm.ws.cdi.extension.spi.test.bundle.extension.MyExtensionString;
+import com.ibm.ws.cdi.extension.spi.test.bundle.getclass.beaninjection.MyBeanInjectionString;
+import com.ibm.ws.cdi.extension.spi.test.bundle.getclass.producer.MyProducedString;
+
+@WebServlet("/CrossWire")
+public class CrossWireTestServlet extends HttpServlet {
```

Comment:
I'd rather new tests were written as `FATServlet` classes ... to do the test in this class rather than just returning a string to be checked elsewhere. However, you might wish to overhaul the FAT bucket as a whole in a follow-up rather than just changing this one test.